pkgs/applications/video/obs-studio/plugins/obs-tuna/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, obs-studio
+, cmake
+, zlib
+, curl
+, taglib
+, dbus
+, pkg-config
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "obs-tuna";
+  version = "1.9.6";
+
+  nativeBuildInputs = [ cmake qtbase ];
+  buildInputs = [ obs-studio zlib curl taglib dbus pkg-config wrapQtAppsHook ];
```

Comment:
are you sure about these ? Shouldn't they belong to nativebuildinputs ?

Reply:
I was actually not sure of the differences until now, I believe a more correct solution would be the following?
```nix
nativeBuildInputs = [ cmake pkg-config wrapQtAppsHook ];
buildInputs = [ obs-studio qtbase zlib curl taglib dbus ];
```

Reply:
I'm not sure yet, I'll ask to a mentor.